qa/L0_perf_nomodel/test.sh

```diff
@@ -79,26 +81,34 @@ TEST_NAMES=(
     "${UNDERTEST_NAME} 16MB I/O Latency HTTP"
     "${UNDERTEST_NAME} Maximum Throughput GRPC"
     "${UNDERTEST_NAME} Maximum Throughput HTTP")
```

Comment:
Here an other places below you left the closing paren here instead of moving it to the end

Reply:
Fixed